draft-ietf-rats-corim.md

```diff
@@ -1183,15 +1183,12 @@ If the search criteria are satisfied, the `endorsements` entries are asserted wi
 
 #### Conditional Endorsement Series Triple {#sec-comid-triple-cond-series}
 
-A Conditional Endorsement Series triple uses a "stateful environment" that identifies a Target Environment plus the measurements that have matching Evidence.	
-
-The series object is an array of `conditional-series-record` that has both Reference and Endorsed Values.
-Each conditional-series-record record is evaluated in the order it appears in the series array.
-The Endorsed Values are accepted if the series condition in a `conditional-series-record` matches the attester's actual state.
-The first `conditional-series-record` that successfully matches an attester's actual state terminates the matching and the corresponding Endorsed Values are accepted.
-If none of the series conditions match the attester's actual state, the triple is not matched, and no Endorsed values are accepted.	
-
-More clarification about the usage and matching order will be resolved by: [^tracked-at] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/321
+The Conditional Endorsement Series Triple is used to assert endorsed values based on an initial condition match followed by a series condition match.
+Every `series-condition-record` selection MUST select the same `mkey`s.
+Every selected `mkey`'s corresponding mentioned set of keys `mval`.key MUST be the same across each `series-condition-record`.
```

Comment:
Though one to parse... I am not sure I get what you mean here.  Is it that _"each selected `mkey` MUST have the same corresponding set of `mval`s across all `series-condition-record` entries"_ ?

Reply:
Say you have 3 measurement-map in a selection. One has no mkey with mval { 0: …}. One has mkey 1 with mval: { 1: …, 4: … } and one has mkey 2 with mval {0: …, 7: …}. That has to be the same selection structure for all selections in the series.

Reply:
OK, surely L1188 needs unpacking.  I suggest reorganising it this way:
1. make a short normative statement, followed by
2. an example